packages/styling/src/styles/GeneralStyles.ts

```diff
@@ -7,4 +7,11 @@ export const normalize: IRawStyle = {
   margin: 0,
   padding: 0,
   boxSizing: 'border-box'
-};
\ No newline at end of file
+};
+
+export const noWrap: IRawStyle = {
+  display: 'block',
```

Comment:
probably would prefer inline-block here, so things don't take up all available width
although i'd prefer just leaving off display altogether and letting the consumer decide between block/inline-block

Reply:
Probably a good idea. Thanks for all your thorough reviewing!